.travis.yml

```diff
@@ -27,12 +27,17 @@ env:
   - DH_REPO="cloudsuite/spark" IMG_TAG="1.5.1" DF_PATH="./commons/spark/1.5.1"
   - DH_REPO="cloudsuite/spark" IMG_TAG="2.1.0" DF_PATH="./commons/spark/2.1.0"
 
+  # Faban
+  - DH_REPO="cloudsuite/faban" IMG_TAG="latest" DF_PATH="./commons/faban/1.3"
```

Comment:
I think it makes sense to have the latest tag point to the actual latest version of faban.

Reply:
We have to be careful with this because when I did a regression test, the newest versions of faban changed their compatibility with java.

If we are going to continually point to the latest version we need to make sure 2 things:
- the link is always updated
- compatibility is always there

I'm fine with either one, but using a specific version is less work

Reply:
Agreed.
For our benchmarks we point to the Faban that is tested and we are sure it works (1.3 for now).